test/helpers/deployer.js

```diff
@@ -169,6 +169,17 @@ class Deployer {
     return this.depositManager
   }
 
+  async deployDrainable() {
+    this.drainable = await contracts.Drainable.new()
+    await this.depositManagerProxy.updateImplementation(this.drainable.address)
+    await this.updateContractMap(
```

Comment:
Why is this required?

Reply:
Just for consistency, isn't a requirement for this. Would you suggest removing it?

Reply:
There is no question of consistency in contracts. Gotta save all storage and gas possible.